bigchaindb/backend/localmongodb/query.py

```diff
@@ -300,11 +300,11 @@ def get_validator_set(conn, height=None):
 
 
 @register_query(LocalMongoDBConnection)
-def get_validator_set_by_election_id(conn, election_id):
+def get_result_by_election_id(conn, election_id, table):
```

Comment:
1. A more appropriate function name could be `get_election_result`/`get_election_result_by_id` ?
2. What is the point of this generalization?

Reply:
1. Good suggestion. Changed the name.

2. When checking the election status, the first thing I want to do is look and see if anything with this `election_id` has been written to the relevant Mongo table, (ie has the vote been previoously concluded.)